tests/test-ui-components/src/labicon.spec.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { expect } from 'chai';
+
+import { LabIcon } from '../..//ui-components/icon';
+
+describe('@jupyterlab/nbformat', () => {
```

Comment:
`ui-components`*

Reply:
Thanks! That was pretty messed up. Fixed